docs/contrib/tests.md

```diff
@@ -0,0 +1,26 @@
+We use `pytest`, `flake8` and `mypy` as a quality gate. We follow `wemake_python_styleguide` to enforce quality.
+
+Since storage bucket names must be globally unique, we use a environment variable for the storage bucket name in tests. which means that you must set the env var `STORAGE_BUCKET_NAME` to whatever the name of your bucket is.
+>This might be removed in the future.
+
+To run all tests:
+
+```
+poetry run pytest
+```
+
+To run linting:
+
+```
+poetry run flake8 .
+```
+
+To check typing:
+
+```
+poetry run mypy .
+```
+
+Default installation of virtual environment is in `.venv`. If you need customize parameters you can do it in `setup.cfg`.
```

Comment:
What is meant with 'if you need customize parameters you can do it in `setup.cfg`?
if people want to install the virtualenv in the project folder it should be in .venv only.
if people want to install other places then thats a thing poetry will handle so i don't think we need to care about it.
The setting `virtualenvs.in-project=false` should make poetry install it in some local poetry folder and activates it automatically.
So I'm for deleting this line.